### ISSUE TITLE
Issue in label dataset of 2018.Lebedev M A, Vizilter Y V, Vygolov O V, et al. Change detection in remote sensing images using conditional adversarial networks

### ISSUE THREAD

**Issue by liuchangf** [wenhwu/awesome-remote-sensing-change-detection#5]
Hello:
        Thank you for sharing resource firstly.
        I found the label data such as 
         "ChangeDetectionDataset\Real\subset\train\OUT\00000.jpg" is not a binary image ，which contain values as following , you can also found it by eyeball 。So it can not be used as organized dataset for training without image preprocess , what is your opinion?

array([  0,   1,   2,   3,   4,   5,   6,   7,   8,   9,  10,  11,  12,
        13,  14,  15,  16,  17,  18,  19,  20,  21,  22,  23,  24,  25,
        27,  28,  29,  30,  31,  32,  34,  35,  36,  38,  39,  40,  41,
        43,  44,  45,  46,  47,  48,  50,  51,  53,  54,  55,  56,  57,
        58,  59,  60,  62,  64,  65,  66,  67,  68,  69,  70,  71,  72,
        73,  74,  75,  76,  78,  79,  80,  81,  83,  84,  85,  86,  88,
        90,  91,  92,  94,  95,  96,  97,  99, 100, 101, 102, 103, 104,
       106, 107, 108, 109, 110, 111, 112, 113, 114, 115, 116, 117, 118,
       119, 121, 122, 123, 124, 125, 127, 128, 129, 130, 131, 132, 133,
       134, 135, 136, 137, 138, 139, 140, 141, 142, 143, 144, 145, 146,
       147, 148, 149, 150, 151, 152, 153, 154, 155, 156, 157, 158, 160,
       161, 162, 163, 164, 165, 166, 167, 168, 169, 170, 171, 172, 173,
       174, 175, 177, 178, 179, 180, 182, 183, 184, 185, 186, 187, 188,
       189, 190, 191, 193, 194, 195, 196, 197, 198, 199, 201, 202, 203,
       204, 205, 206, 207, 208, 209, 210, 212, 214, 215, 217, 218, 219,
       220, 221, 222, 223, 224, 225, 226, 227, 228, 229, 230, 231, 232,
       233, 234, 235, 236, 237, 238, 240, 241, 242, 243, 244, 245, 246,
       247, 248, 249, 251, 252, 253, 254, 255], dtype=uint8)

![图片](https://user-images.githubusercontent.com/31433358/82771651-50a88000-9e6f-11ea-9c4e-3a7a150a2ecb.png)

**Comment by wenhwu** [comment]
Thank you for providing such useful information, and maybe you found something interesting.

I used `ENVI's Quick stats` and `numpy.unique` to analyze the dataset and observed the same phenomenon. In fact, the training set, validation set, and test set all have labels that are not binary maps (note, the main values in the labels are still 0 and 255). To me, maybe this is a mistake.

[daifeng2016/End-to-end-CD-for-VHR-satellite-image](https://github.com/daifeng2016/End-to-end-CD-for-VHR-satellite-image) used this dataset, I have already opened an issue in [The label of the dataset is not binary map](https://github.com/daifeng2016/End-to-end-CD-for-VHR-satellite-image/issues/11), maybe you can take a look.

If this dataset is not necessary, I recommend you use the [LEVIR-CD](https://justchenhao.github.io/LEVIR/) dataset. Or you can email the author of the dataset for advice, the paper of the dataset is this [URL](https://www.int-arch-photogramm-remote-sens-spatial-inf-sci.net/XLII-2/565/2018/isprs-archives-XLII-2-565-2018.pdf).

Please correct me if I made any mistakes.

**Comment by Bobholamovic** [comment]
Hello, here is another question on this dataset. I downloaded the compressed file and found that the validation set contained only 2998 image pairs, which contradicts the "3000 pairs for validation" as mentioned in the paper. I wonder if I've had the complete file downloaded. Do you encounter the same problem?

**Comment by wenhwu** [comment]
@Bobholamovic
Thank you for providing this detail. Yes, I also only have 2,998 image pairs. I do not recommend this dataset because of the problem mentioned above.
